src/Equinor.ProCoSys.IPO.Command/Validators/InvitationValidators/IInvitationValidator.cs

```diff
@@ -11,22 +11,25 @@ public interface IInvitationValidator
         bool IsValidScope(DisciplineType type, IList<string> mcPkgScope, IList<string> commPkgScope);
         Task<bool> ParticipantWithIdExistsAsync(ParticipantsForCommand participant, int invitationId, CancellationToken token);
         Task<bool> ParticipantExistsAsync(int id, int invitationId, CancellationToken token);
+        Task<bool> IsSignedParticipantAsync(int id, int invitationId, CancellationToken token);
```

Comment:
Is IsSignerParticipantAsync a better name?

Reply:
This method checks if a participant is signed, but I can rename to ParticipantIsSignedAsync instead